packages/react/src/components/select/Select.module.scss

```diff
@@ -48,6 +57,19 @@
     padding: var(--spacing-s);
     padding-right: var(--spacing-2-xs);
 
+    &.withVisibleFocus {
+      align-self: center;
+      height: calc(calc(var(--dropdown-height)) - 6px);
```

Comment:
Is this 6px just by eye-testing, could it be like half of something or? If some other height changes this will break?

Reply:
<img width="448" alt="image" src="https://github.com/user-attachments/assets/77e92284-2018-40d2-90da-17056784b1e4">

The `-6px` makes the focus area of the clear button to fit inside the dropdown button area, which is `--dropdown-height`.

Nothing else is currently affecting the relationship, except if the border sizes change in the future.